File: intelligencecity_user_web/src/main/webapp/assets/js/service/commonuserService.js
//服务层
app.service('commonuserService',function($http){
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../commonuser/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../commonuser/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../commonuser/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		alert("已经到了servicec层的代码")
		return  $http.post('../commonuser/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../commonuser/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../commonuser/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../commonuser/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: intelligencecity_manager_interface/src/main/java/com/shanghaiuniversity/manager/service/CommonuserService.java
package com.shanghaiuniversity.manager.service;

import com.ShanghaiUniversity.entity.PageResult;

import java.util.List;

/**
 * 服务层接口
 *
 * @author Administrator
 */
public interface CommonuserService {

    /**
     * 返回全部列表
     *
     * @return
     */
    public List<Commonuser> findAll();

    /**
     * 返回分页列表
     *
     * @return
     */
    public PageResult findPage(int pageNum, int pageSize);

    /**
     * 增加
     */
    public void add(Commonuser commonuser);

    /**
     * 修改
     */
    public void update(Commonuser commonuser);

    /**
     * 根据ID获取实体
     *
     * @param id
     * @return
     */
    public Commonuser findOne(int id);

    /**
     * 批量删除
     *
     * @param ids
     */
    public void delete(int[] ids);

    /**
     * 分页
     *
     * @param pageNum  当前页 码
     * @param pageSize 每页记录数
     * @return
     */
    public PageResult findPage(Commonuser commonuser, int pageNum, int pageSize);

}
